kit/src/components/user_image_group/mod.rs

```diff
@@ -27,12 +27,11 @@ pub fn emit(cx: &Scope<Props>, e: Event<MouseData>) {
 #[allow(non_snake_case)]
 pub fn UserImageGroup<'a>(cx: Scope<'a, Props<'a>>) -> Element<'a> {
     let pressable = cx.props.onpress.is_some();
-    let count = cx.props.participants.len() as i64 - 3;
+    let count = cx.props.participants.len() as i64 - 3; // todo: why is this  - 3?
     let group = cx.props.participants.len() > 2;
     let username = cx.props.with_username.clone().unwrap_or_default();
-    let single_user = &cx.props.participants[0];
 
-    let loading = cx.props.loading.unwrap_or_default();
+    let loading = cx.props.loading.unwrap_or_default() || cx.props.participants.is_empty();
```

Comment:
Looks good for now but wondering if we can have a better process, maybe hoisting this since it uses the loading prop.